frontend/src/components/app.jsx

```diff
@@ -3,41 +3,12 @@
 
 import React from 'react/addons';
 import {RouteHandler, Navigation} from 'react-router';
```

Comment:
There is no point in having Navigation then, since you removed `this.transitionTo`